msteams-platform/resources/schema/manifest-schema-dev-preview.md

```diff
@@ -955,14 +955,15 @@ The `extensions` property specifies Outlook Add-ins within an app manifest and s
 |`autoRunEvents`| Array | | | Defines the event-based activation extension point. |
 |`alternates`| Array | | | Specifies the relationship to alternate existing Microsoft 365 solutions. It's used to hide or prioritize add-ins from the same publisher with overlapping functionality. |
 |`audienceClaimUrl`| String | 2048 characters | | Specifies the URL for your extension and is used to validate Exchange user identity tokens. For more information, see [inside the Exchange identity token](/office/dev/add-ins/outlook/inside-the-identity-token)|
+|`contextMenus`| Array | | | Specifies the context menus for your extension. A context menu is a shortcut menu that appears when you select and hold (or right-click) in the Office UI. |
```

Comment:
Is "Office UI" a specific concept, or is it meant here to stand for the UI of the given WXPO host app? If the latter, how about something like `UI of the host application`? /Cc @v-kushals

Reply:
The right click will always be in the open Office document. They won't be right clicking the Office ribbon or anything else in the Office UI, so I would recommend replacing "in the Office UI" with "in the open Office document".